README.md

```diff
@@ -1,4 +1,35 @@
-# iroha2-block-explorer-backend
+# Iroha 2 Blockchain Explorer Backend
+
+This readme provides an overview of the Iroha 2 blockchain explorer backend and instructions on how to install, run, and utilize the explorer's backend features.
+
+## Installation
+
+To set up the Iroha 2 blockchain explorer backend, follow these steps:
+
+1. Install Rust.
+2. For full functionality of the explorer backend, build [Iroha `v2.0.0-pre-rc.9`](https://github.com/hyperledger/iroha/tree/ea45b5053018acd48340024800786ff5a3d0904d) and ensure it is running.
```

Comment:
only this release of Iroha or any later releases are also ok?

it's probably a good idea to refer to the build instructions https://hyperledger.github.io/iroha-2-docs/guide/build.html

Reply:
I tested it with the latest Iroha version and it works fine.

Reply:
I tested it out with the latest version of Iroha but it didn't work! 

![Screenshot 2023-06-21 002502](https://github.com/soramitsu/iroha2-block-explorer-backend/assets/76839614/d7e882e7-4a40-47e3-b84b-d2698da5aab7)